### PR TITLE
feat(): delete contacted person

### PR DESCRIPTION
**Pull request by yaniryos**
Added Schemas changes:
https://github.com/CoronaTeamMOD/CoronaISchemas/pull/1

### CODE REVIEW

**Review by sandymash — DISMISSED** [review]
שונאת אותך

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/ContactDateCard/InteractionCard/ContactGrid/ContactGridStyles.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+
```
Comment (roymaniv) [inline client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/ContactDateCard/InteractionCard/ContactGrid/ContactGridStyles.ts:1]: למחוק

File: server/src/DBService/ContactEvent/Mutation.ts [hunk 2]
```diff
@@ -22,4 +22,12 @@ mutation deleteContactEvent ($contactEventId: Int!) {
     clientMutationId
   }
 }   
+`;
+
+export const DELETE_CONTCTED_PERSON = gql`
```
Comment (roymaniv) [inline server/src/DBService/ContactEvent/Mutation.ts:27]: CONTCTED --> CONTACTED

**Review by roymaniv — APPROVED** [review]

**Review by roymaniv — APPROVED** [review]

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionEventForm/useInteractionsForm.ts [hunk 3]
```diff
@@ -33,6 +35,10 @@ const useInteractionsForm = (props : useInteractionFormIncome): useInteractionFo
           ...interactionsDataToSave,
           [InteractionEventDialogFields.LOCATION_ADDRESS]: locationAddress,
           [InteractionEventDialogFields.INVESTIGATION_ID]: epidemiologyNumber,
+          contacts: interactionsDataToSave.contacts.map((contact: Contact) => ({
```
Comment (ItayBenMoshe) [inline client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionEventForm/useInteractionsForm.ts:38]: use InteractionEventDialogFields to get 'contacts'

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionEventForm/useInteractionsForm.ts [hunk 4]
```diff
@@ -68,7 +74,11 @@ const useInteractionsForm = (props : useInteractionFormIncome): useInteractionFo
         axios.post('/intersections/createContactEvent', {
           ...interactionsDataToSave,
           [InteractionEventDialogFields.LOCATION_ADDRESS]: locationAddress,
-          [InteractionEventDialogFields.INVESTIGATION_ID]: epidemiologyNumber
+          [InteractionEventDialogFields.INVESTIGATION_ID]: epidemiologyNumber,
+          contacts: interactionsDataToSave.contacts.map((contact: Contact) => ({
```
Comment (ItayBenMoshe) [inline client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionEventForm/useInteractionsForm.ts:78]: same as above

**Review by roymaniv — APPROVED** [review]

**Review by ItayBenMoshe — APPROVED** [review]

**Pull request merged**